settings/speakcivi.setting.php

```diff
@@ -273,6 +273,36 @@
     'description' => "From",
     'help_text' => "From",
   ),
+  'activity_type_join' => array(
```

Comment:
sorry, I wasn't clear, I was suggesting to simply create the activities as a managed entity (ie. be sure join and leave exists), not to let the option to choose